### PR TITLE
Upgrade components to use new env with Ubuntu 22.04

### PR DESCRIPTION
**Pull request by ashg1910**
Loop page with links to sanity, benchmarking runs, results comparison with older env and more details - [Ubuntu base image upgrade](https://loop.cloud.microsoft/p/eyJ3Ijp7InUiOiJodHRwczovL21pY3Jvc29mdC5zaGFyZXBvaW50LmNvbS8%2FbmF2PWN6MGxNa1ltWkQxaUlWazFXRE5MVmtrMFNWVnBiekpGVTJwclREZG9hRE5HUWxkaldURXpaamxDYm5WQ1NIUm5kMUozU0Y5blh6WnlOaTFyVjNOUmIweHRkM3BYYkV4MlMwb21aajB3TVVsSE5rdFdNazVRVmtOS1dWaFRXVlZPU2toTFFWcFBSbGhRU0U4MVFWZEZKbU05Sm1ac2RXbGtQVEUlM0QiLCJyIjpmYWxzZX0sInAiOnsidSI6Imh0dHBzOi8vbWljcm9zb2Z0LnNoYXJlcG9pbnQuY29tLzpmbDovci9zaXRlcy83ZmZlYjFlZC05YjFmLTQwMWUtOGM0Yi0yZGNkZWEyOWRjYzgvU2hhcmVkJTIwRG9jdW1lbnRzL0xvb3BBcHBEYXRhL1VudGl0bGVkJTIwMjMubG9vcD9kPXc0NjhhMjIzODQ3ZDY0YzA0OWUxM2IxYTFlZDkxMGQ2YiZjc2Y9MSZ3ZWI9MSZuYXY9Y3owbE1rWnphWFJsY3lVeVJqZG1abVZpTVdWa0xUbGlNV1l0TkRBeFpTMDRZelJpTFRKa1kyUmxZVEk1WkdOak9DWmtQV0loV1RWWU0wdFdTVFJKVldsdk1rVlRhbXRNTjJob00wWkNWMk5aTVRObU9VSnVkVUpJZEdkM1VuZElYMmRmTm5JMkxXdFhjMUZ2VEcxM2VsZHNUSFpMU2labVBUQXhTVWMyUzFZeVNsbEZTMFpGVGxaVFNFRlNSMG8wUlRWU1ZVaFhXa05FVEV3bVl6MGxNa1ltWm14MWFXUTlNU1poUFV4dmIzQkJjSEFtY0QwbE5EQm1iSFZwWkhnbE1rWnNiMjl3TFhCaFoyVXRZMjl1ZEdGcGJtVnlKbmc5SlRkQ0pUSXlkeVV5TWlVelFTVXlNbFF3VWxSVlNIaDBZVmRPZVdJelRuWmFibEYxWXpKb2FHTnRWbmRpTW14MVpFTTFhbUl5TVRoWmFVWmFUbFpuZWxNeFdrcE9SV3hXWVZjNGVWSldUbkZoTUhjellVZG5lbEpyU2xoWk1XdDRUVEpaTlZGdE5URlJhMmd3V2pOa1UyUXdhR1phTVRneVkycFpkR0V4WkhwVlZ6bE5ZbGhrTmxZeWVFMWthM1JMWmtSQmVGTlZZekpUTVZsNVZHeENWMUV3Y0ZwWFJrNWFWbFUxUzFORmRFSlhhemxIVjBaQ1NWUjZWa0pXTUZVbE0wUWxNaklsTWtNbE1qSnBKVEl5SlROQkpUSXlabU13TkRaa1ptTXROV1prTnkwMFlqSXlMVGhpWW1JdFpqZ3hObVU1T0RaaE1HTmpKVEl5SlRkRSIsInIiOmZhbHNlfSwiaSI6eyJpIjoiZmMwNDZkZmMtNWZkNy00YjIyLThiYmItZjgxNmU5ODZhMGNjIn19)

### CODE REVIEW

**Comment by ashg1910** [comment]
Loop page with links to sanity, benchmarking runs, results comparison with older env and more details - [Ubuntu base image upgrade](https://loop.cloud.microsoft/p/eyJ3Ijp7InUiOiJodHRwczovL21pY3Jvc29mdC5zaGFyZXBvaW50LmNvbS8%2FbmF2PWN6MGxNa1ltWkQxaUlWazFXRE5MVmtrMFNWVnBiekpGVTJwclREZG9hRE5HUWxkaldURXpaamxDYm5WQ1NIUm5kMUozU0Y5blh6WnlOaTFyVjNOUmIweHRkM3BYYkV4MlMwb21aajB3TVVsSE5rdFdNazVRVmtOS1dWaFRXVlZPU2toTFFWcFBSbGhRU0U4MVFWZEZKbU05Sm1ac2RXbGtQVEUlM0QiLCJyIjpmYWxzZX0sInAiOnsidSI6Imh0dHBzOi8vbWljcm9zb2Z0LnNoYXJlcG9pbnQuY29tLzpmbDovci9zaXRlcy83ZmZlYjFlZC05YjFmLTQwMWUtOGM0Yi0yZGNkZWEyOWRjYzgvU2hhcmVkJTIwRG9jdW1lbnRzL0xvb3BBcHBEYXRhL1VudGl0bGVkJTIwMjMubG9vcD9kPXc0NjhhMjIzODQ3ZDY0YzA0OWUxM2IxYTFlZDkxMGQ2YiZjc2Y9MSZ3ZWI9MSZuYXY9Y3owbE1rWnphWFJsY3lVeVJqZG1abVZpTVdWa0xUbGlNV1l0TkRBeFpTMDRZelJpTFRKa1kyUmxZVEk1WkdOak9DWmtQV0loV1RWWU0wdFdTVFJKVldsdk1rVlRhbXRNTjJob00wWkNWMk5aTVRObU9VSnVkVUpJZEdkM1VuZElYMmRmTm5JMkxXdFhjMUZ2VEcxM2VsZHNUSFpMU2labVBUQXhTVWMyUzFZeVNsbEZTMFpGVGxaVFNFRlNSMG8wUlRWU1ZVaFhXa05FVEV3bVl6MGxNa1ltWm14MWFXUTlNU1poUFV4dmIzQkJjSEFtY0QwbE5EQm1iSFZwWkhnbE1rWnNiMjl3TFhCaFoyVXRZMjl1ZEdGcGJtVnlKbmc5SlRkQ0pUSXlkeVV5TWlVelFTVXlNbFF3VWxSVlNIaDBZVmRPZVdJelRuWmFibEYxWXpKb2FHTnRWbmRpTW14MVpFTTFhbUl5TVRoWmFVWmFUbFpuZWxNeFdrcE9SV3hXWVZjNGVWSldUbkZoTUhjellVZG5lbEpyU2xoWk1XdDRUVEpaTlZGdE5URlJhMmd3V2pOa1UyUXdhR1phTVRneVkycFpkR0V4WkhwVlZ6bE5ZbGhrTmxZeWVFMWthM1JMWmtSQmVGTlZZekpUTVZsNVZHeENWMUV3Y0ZwWFJrNWFWbFUxUzFORmRFSlhhemxIVjBaQ1NWUjZWa0pXTUZVbE0wUWxNaklsTWtNbE1qSnBKVEl5SlROQkpUSXlabU13TkRaa1ptTXROV1prTnkwMFlqSXlMVGhpWW1JdFpqZ3hObVU1T0RaaE1HTmpKVEl5SlRkRSIsInIiOmZhbHNlfSwiaSI6eyJpIjoiZmMwNDZkZmMtNWZkNy00YjIyLThiYmItZjgxNmU5ODZhMGNjIn19)

**Review by Anubha98 — APPROVED** [review]

File: assets/training/finetune_acft_hf_nlp/components/pipeline_components/question_answering/spec.yaml
Comment (ashg1910) [reply assets/training/finetune_acft_hf_nlp/components/pipeline_components/question_answering/spec.yaml:684]: yes 0.0.35 isn't released yet. I will merge this only once eval components are merged.
Don't want to make multiple component releases thats why added this change here.

**Pull request merged**
